[ADD] account_type_multi_company

sbidoul opened the pull request.

sbidoul committed:
[ADD] account_type_multi_company

pedrobaeza commented on account_type_multi_company/__openerp__.py:
```diff
@@ -0,0 +1,43 @@
+              "Noviat,"
+              "Odoo Community Association (OCA)",
+    'category': 'Accounting & Finance',
+    'version': '8.0.1.0',
```
Use 8.0.1.0.0

pedrobaeza commented on account_type_multi_company/README.rst:
```diff
@@ -0,0 +1,60 @@
+
+ * N/A
+
+Credits
```
The bug tracker section is missing

sbidoul commented:
@pedrobaeza argh. I forgot to git pull our scaffold template before creating the module.
I updated the README and manifest. Also added the runbot link.
Thanks for the review.

pedrobaeza commented on account_type_multi_company/README.rst:
```diff
@@ -0,0 +1,74 @@
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/account_type_multi_company/issues>`_.
```
s/account_type_multi_company/multi-company

pedrobaeza commented on account_type_multi_company/README.rst:
```diff
@@ -0,0 +1,74 @@
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/account_type_multi_company/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/multi_company/issues/new?body=module:%20account_type_multi_company%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
s/multi_company/multi-company

sbidoul commented:
@pedrobaeza Oops. Done.

pedrobaeza commented:
:+1: better if you squash commits in one as there are little changes between them

luc-demeyer commented:
:+1:

lmignon commented on account_type_multi_company/models/account_type.py:
```diff
@@ -0,0 +1,33 @@
+class AccountAccountType(models.Model):
+    _inherit = 'account.account.type'
+
+    active = fields.Boolean(string='Active', select=True, default=True)
```
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#field 

> If the technical name of the field (the variable name) is the same to the string of the label, don't put string parameter for new API fields, because it's automatically taken. If your variable name contains "_" in the name, they are converted to spaces when creating the automatic string and each word is capitalized. (example: old api 'name': fields.char('Name', ...) new api 'name': fields.Char(...))

lmignon commented:
:+1: One minor comment.

sbidoul commented:
@lmignon okay, done.

Should be good for merging, now.
